tests/packagedcode/data/opam/sample3/output.opam.expected

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "opam",
+  "namespace": null,
+  "name": null,
+  "version": null,
+  "qualifiers": {},
+  "subpath": null,
+  "primary_language": "Ocaml",
+  "description": "Interactive theorem prover based on lambda-tree syntax",
+  "release_date": null,
+  "parties": [
+    {
+      "type": "person",
+      "role": "author",
+      "name": "Andrew Gacek",
+      "email": null,
+      "url": null
+    },
+    {
+      "type": "person",
+      "role": "author",
+      "name": "Yuting Wang",
+      "email": null,
+      "url": null
+    },
+    {
+      "type": "person",
+      "role": "author",
+      "name": "Kaustuv Chaudhuri",
+      "email": null,
+      "url": null
+    },
+    {
+      "type": "person",
+      "role": "maintainer",
+      "name": null,
+      "email": "kaustuv@chaudhuri.info",
+      "url": null
+    }
+  ],
+  "keywords": [],
+  "homepage_url": "http://abella-prover.org",
+  "download_url": null,
+  "size": null,
+  "sha1": null,
+  "md5": null,
+  "sha256": null,
+  "sha512": null,
+  "bug_tracking_url": null,
+  "code_view_url": null,
+  "vcs_url": null,
+  "copyright": null,
+  "license_expression": "gpl-3.0",
+  "declared_license": "GPL-3.0-only",
+  "notice_text": null,
+  "root_path": null,
+  "dependencies": [
+    {
+      "purl": "pkg:opam/ocaml",
+      "requirement": ">= 4.01.0",
+      "scope": "dependency",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false
+    },
+    {
+      "purl": "pkg:opam/ocamlfind",
+      "requirement": "",
+      "scope": "dependency",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false
+    },
+    {
+      "purl": "pkg:opam/ocamlbuild",
+      "requirement": "build",
+      "scope": "dependency",
+      "is_runtime": true,
+      "is_optional": false,
+      "is_resolved": false
+    }
+  ],
+  "contains_source_code": null,
+  "source_packages": [],
+  "purl": null,
+  "repository_homepage_url": null,
+  "repository_download_url": null,
+  "api_data_url": null
```

Comment:
Would this be a data API url of sorts?
https://github.com/ocaml/opam-repository/blob/master/packages/winsvc/winsvc.1.0.0/opam
e.g. https://github.com/ocaml/opam-repository/blob/master/packages/{name}/{name}.{version}/opam